### ISSUE TITLE
Convert Inception21k to PyTorch

### ISSUE THREAD

**Issue by rtaori** [dmlc/mxnet-model-gallery#33]
Hi,

I am trying to convert the Inception21k model to PyTorch. I used the [MMdnn tool](https://github.com/Microsoft/MMdnn) to do the conversion; however, I am only seeing 65% top1 accuracy on the validation set (below the reported 68%). 

What was the image preprocessing used to train the network? Currently I am doing:
1) Resize shortest size of image to 256
2) Center crop to 224
3) Multiply by 255 to scale image input range to [0, 255]
4) Subtract mean of 117 from the image

Is this correct?
